src/sancus_support/sm_support.h

```diff
@@ -580,6 +586,20 @@ always_inline sm_id sancus_get_caller_id(void)
     return ret;
 }
 
+// always_inline sm_id sancus_get_original_caller_id(struct SancusModule* sm)
```

Comment:
not sure if this is for future, but let's remove this better altogether, cause the `struct SancusModule` is typically stored and managed in *untrusted* code, so not to be trusted

Reply:
No, sorry that was my local draft that made it into the commit when I blindly added the sancus_disable comments. It's already gone in my code and will be removed in the next commit.